src/components/step-flow/step-flow.test.tsx

```diff
@@ -48,211 +48,209 @@ function calculateStepStateIndexes(
 }
 
 describe("Step Flow component", () => {
-  describe("prop checks", () => {
-    it("when the 'category' prop is passed, the correct element and text renders", () => {
+  it("should render the correct element and text when the category prop is passed", () => {
+    render(
+      <StepFlow
+        title="foo"
+        currentStep={5}
+        totalSteps={6}
+        category="bar"
+        ref={() => {}}
+      />
+    );
+
+    expect(screen.getByText("bar")).toBeVisible();
+  });
+
+  it("when the 'title' prop is passed, the correct element and text renders", () => {
```

Comment:
suggestion(non-blocking): language still wrong on this test

Might be worth adding to the re-visit ticket rather than dealing with it now unless there were other changes to be made